frontend/src/app/components/ExperimentTourDialog/index.js

```diff
@@ -54,17 +54,17 @@ export default class ExperimentTourDialog extends React.Component {
       return <h3 className="modal-header lighter">{experiment.title}</h3>;
     }
     return (<Localized id="tourOnboardingTitle" $title={experiment.title}>
-              <h3 className={cn('modal-header lighter', {
-                enabled: isExperimentEnabled({ addon_id: `@${experiment.title}` })
-              })}>{experiment.title}</h3>
+            <h3 className={cn('modal-header lighter', {
+              enabled: isExperimentEnabled({ addon_id: `@${experiment.title}` })
+            })}>{experiment.title}</h3>
             </Localized>);
   };
 
   renderStep = (tourSteps: Array<Object>, currentStep: number) => {
     return tourSteps.map((step, idx) => (idx === currentStep) && (
         <div key={idx} className="step-content">
           <div className="step-image">
-            <img src={step.image} width="580" height="332"/>
+            <img src={step.image} />
```

Comment:
Looks like this change will cause some reflows & size jumps as things load. Are these images all different sizes? Could we find a way to keep a consistent size here?

Reply:
images are all the same size....i dropped the w and h in in a previous fix. this should be added here.